src/views/OverviewView.js

```diff
@@ -1,25 +1,53 @@
-import { OrthographicView } from '@deck.gl/core';
+/* eslint-disable max-classes-per-file */
+import { Controller, OrthographicView } from '@deck.gl/core';
 import VivView from './VivView';
 import { OverviewLayer } from '../layers';
 import { makeBoundingBox, getVivId } from './utils';
 
+class OverviewState {
+}
+
+class OverviewController extends Controller {
+  constructor(props) {
+    super(OverviewState, props);
+    this.events = ["click"];
+  }
+
+  handleEvent(event) {
+    if (event.type !== "click") {
```

Comment:
I think this would fail prettier with the double-quotes - could you enable GH actions in your fork repo while we resolve the fact that forking PR's don't run actions (#333)?

Reply:
I have a quotes rule in my `.eslintrc.json`: `"quotes": [2, "double", "avoid-escape"]`. May I add that rule, just with single quotes, to yours?

Reply:
@andreasg123 Do you have an `eslint` file overriding ours?  I am a little confused since our `.eslintrc.yml` extends prettier.

Reply:
No, I talked about a different repo. This was just a suggestion how to enforce single quotes but just using prettier is fine